overlord/configstate/config/transaction.go

```diff
@@ -31,6 +31,14 @@ import (
 	"github.com/snapcore/snapd/overlord/state"
 )
 
+// Conf is an interface describing both state and transaction.
+type Conf interface {
+	Get(snapName, key string, result interface{}) error
+	Set(snapName, key string, value interface{}) error
+	Changes() []string
+	State() *state.State
+}
+
```

Comment:
I don't understand this move, as far as I can tell the only place using this is still configcore, were you planning something else?

Reply:
This is so that I can move the helper for getting the state of a feature flag and not end up with huge changes across tests and code _OR_ end up with cyclic imports.

Reply:
Ok, I see, it's because GetFeatureFlag is used in handleExperimentalFlags, this definition should probably go closer to the consumer itself, GetFeatureFlag

Reply:
Done